test/aionlib.crypto.js

```diff
@@ -0,0 +1,44 @@
+var chai = require('chai');
+var assert = chai.assert;
+var hashutil = require('../packages/aion-lib/src/crypto.js');
+
+var inputVal = '123vnfsgj364hbdfh';
+var inputVal2 = '00000000000000000000000000000000';
+var blake128ResLength = 16;
+var blake256ResLength = 32; 
+
+describe('blake2b128', function () {
```

Comment:
can we make it so that the 'should hash correctly' tests assert against the actual value of the blake128 hash (not just the length)?  then we can remove 'should hash consistently'

Reply:
got it